lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -53,13 +50,18 @@ export function contentItemForContent(ltiLaunchURL, content, extraParams) {
       break;
   }
 
+  const queryParams = new URLSearchParams();
+  Object.entries(params).forEach(([key, value]) =>
+    queryParams.append(key, value)
+  );
```

Comment:
nb. The issue the client ran into with repeated query parameters doesn't affect us here. We only ever pass single-valued query params (ie. not arrays).